recipes/head_node_slurm_finalize.rb

```diff
@@ -71,15 +71,15 @@
     require 'chef/mixin/shell_out'
     require 'shellwords'
 
-    # $ /opt/slurm/bin/squeue --name='parallelcluster-init-cluster' -O state -h
+    # $ #{node['cfncluster']['slurm']['install_dir']}/bin/squeue --name='parallelcluster-init-cluster' -O state -h
```

Comment:
minor: I think we don't need to change it in the comment. Because in the comment we are running linux command, which is not related to chef node attribute.